OMCompiler/Makefile.omdev.mingw

```diff
@@ -97,7 +96,7 @@ include Makefile.common
 include Makefile.omsi.common
 
 # For Windows build static FMI lib!
-FMILIB_SHARED = OFF
+FMILIB_SHARED = ON
```

Comment:
@adrpo There was probably a realy good reason for disabling FMI Library to be build as shared library on Windows systems. Do you kow why and if it's still needed?

Reply:
I don't remember why I did this. You can try with it ON and see if is fine.